src/__experimental__/components/date-range/date-range.component.js

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from 'i18n-js';
+import { assign } from 'lodash';
+import classNames from 'classnames';
+import Date from '../date';
+import DateRangeValidator from '../../../utils/validations/date-range';
+import DateHelper from '../../../utils/helpers/date';
+import tagComponent from '../../../utils/helpers/tags';
+import StyledDateRange from './date-range.style';
+
+class DateRange extends React.Component {
+  static propTypes = {
```

Comment:
Can we change it to
`DateRange.propTypes = {}` and move it to bottom of the component

Reply:
we have agreed to not refactor the component in this task